src/components/Calculator.js

```diff
@@ -0,0 +1,43 @@
+import React from 'react';
+import Button from './button';
+import Result from './result';
+import './css/styles.css';
+
+const MyCalc = () => (
+  <div className="frame">
+    <Result />
+    <div className="autput">
+      <div className="buttons">
+        <Button label="AC" />
+        <Button label="+/-" />
+        <Button label="%" />
+        <Button label="/" className="divide" />
+      </div>
+      <div className="buttons">
+        <Button label="7" />
+        <Button label="8" />
+        <Button label="9" />
+        <Button label="*" className="dot" />
+      </div>
+      <div className="buttons">
+        <Button label="4" />
+        <Button label="5" />
+        <Button label="6" />
+        <Button label="-" className="minus" />
+      </div>
+      <div className="buttons">
+        <Button label="1" />
+        <Button label="2" />
+        <Button label="3" />
+        <Button label="+" className="plus" />
+      </div>
+      <div className="buttons">
+        <Button label="0" className="zero" />
+        <Button label="." />
+        <Button label="=" className="equals" />
+      </div>
+    </div>
+  </div>
```

Comment:
- [ ] It would be better to follow the design, please make the Screen for displaying input take the whole space at the top and look like the design, this will make your design look like the design template 👍🏿 

 Design          |  Your design     
:-------------------------:|:-------------------------:
<img width="558" alt="calculator" src="https://user-images.githubusercontent.com/69638013/228359721-dff89ad5-a7e2-4f2d-b226-3cb7fa107e52.png"> |   ![Screenshot_795](https://user-images.githubusercontent.com/69638013/228359737-f3acc654-fb15-4465-bb79-31979328a74e.png)